<?xml version="1.0"?>
<launch>
  <!-- pose_initializer -->
  <group>
    <include file="$(find-pkg-share pose_initializer)/launch/pose_initializer.launch.xml">
      <arg name="gnss_enabled" value="true"/>
      <arg name="ndt_enabled" value="true"/>
      <arg name="ekf_enabled" value="true"/>
      <arg name="stop_check_enabled" value="true"/>
    </include>
  </group>

  <!-- pose_initializer (Automatic call from AD API) -->
  <group>
    <include file="$(find-pkg-share autoware_automatic_pose_initializer)/launch/automatic_pose_initializer.launch.xml"/>
  </group>

</launch>
